src/componentsEventPage/EventForm.js

```diff
@@ -0,0 +1,272 @@
+                    errors.eventEndDate && errors.eventEndDate.type === 'required' &&
+                    (
+                        <ErrorMessage>Date is required!</ErrorMessage>
+                    )}
+
+                <StyledInputLabel>
+                    Address:
+                   <StyledInput
+                        name="street"
+                        ref={register({ required: true, minLength: 5 })}
+                    />
+                </StyledInputLabel>
+
+                {errors.street && errors.street.type === 'required' && (
+                    <ErrorMessage>Street is required!</ErrorMessage>
+                )}
+                {errors.street && errors.street.type === 'minLength' && (
+                    <ErrorMessage>
+                        This field requires at least 5 characters!
+                    </ErrorMessage>
+                )}
+
+                <StyledInput
+                    name="zip"
+                    ref={register({ required: true, minLength: 2 })}
+                />
+
+                {errors.zip && errors.zip.type === 'required' && (
+                    <ErrorMessage>Zip is required!</ErrorMessage>
+                )}
+                {errors.zip && errors.zip.type === 'minLength' && (
+                    <ErrorMessage>
+                        This field requires at least 2 characters!
+                    </ErrorMessage>
+                )}
+
+                <StyledInputLabel>
+                    Website:
+          <StyledInput
+                        name="website"
+                        ref={register({ required: true, minLength: 8 })}
+                    />
+                </StyledInputLabel>
+
+                {errors.website && errors.website.type === 'required' && (
+                    <ErrorMessage>Website is required!</ErrorMessage>
+                )}
+                {errors.website && errors.website.type === 'minLength' && (
+                    <ErrorMessage>
+                        This field requires at least 8 characters!
+                    </ErrorMessage>
+                )}
+
+                <StyledInputLabel>
+                    TicketPrice:
+                    <StyledInput
+                        name="price"
+                        ref={register({ required: false })}
+                    />
+                </StyledInputLabel>
+
+                <StyledInputLabel>
+                    Picture:
+                   <StyledInput
+                        name="poster"
+                        ref={register({ required: false })}
+                    />
+                </StyledInputLabel>
+
+                <ButtonGroup>
+                    <button onClick={onCancel} type="button">
+                        Cancel
+                    </button>
+                    <SubmitButton type="submit">Save</SubmitButton>
+                </ButtonGroup>
+            </Form>
+        </>
+    )
+}
+
+const ErrorMessage = styled.p`
+  color: var(--orange-main);
```

Comment:
Var is somewhat outdated- would you consider using 'let' instead? 
I found the article balanced and helpful. 
https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/